hadoop-gremlin/src/test/java/org/apache/tinkerpop/gremlin/hadoop/HadoopGraphFeatureIntegrateTest.java

```diff
@@ -75,6 +77,9 @@
         add(Pair.with("g_V_matchXa_followedBy_count_isXgtX10XX_b__a_0followedBy_count_isXgtX10XX_bX_count", skipReasonLength));
     }};
 
+    private static final List<String> TAGS_TO_IGNORE = Arrays.asList(
```

Comment:
do we need to define it this way? any reason to not just add this tag to the `CucumberOptions` annotation above?